### DISCUSSION TITLE
Welcome to WikiGen Discussions!

### DISCUSSION THREAD

**Discussion by AkeemAllen** [AkeemAllen/WikiGen#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsmithcm10** [comment]
Hey man. Just started using your tool and very excited about it. I had a quick question about its capabilites. I am making a hoenn hack with a specific regional dex. When generating pokemon, they seem to always go in national dex order. Is there a way to implement them in a regional dex format? Or just making each of them "custom" and creating the dex order that way? Thanks in advance.
